syz-cluster/email-reporter/main.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2025 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+// NOTE: This app assumes that only one copy of it is runnning at the same time.
+
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/google/syzkaller/pkg/email"
+	"github.com/google/syzkaller/syz-cluster/pkg/app"
+)
+
+// TODO: add extra sanity checks that would prevent flooding the mailing lists:
+// - this pod may crash and be restarted by K8S: this complicates accounting,
+// - the send operation might return an error, yet an email would be actually sent: back off on errors?
+
+// How often to check whether there are new emails to be sent.
+const pollPeriod = 30 * time.Second
```

Comment:
And we'll probably need the cluster confuguration management system. I'm sure we can reuse something existing.

Reply:
There's https://github.com/google/syzkaller/blob/master/syz-cluster/pkg/app/config.go that got further extended in this PR, but I think we should only add new values there when we're sure that we're going to need to change them (especially between dev and prod envs). Otherwise we'll just drown in the amount of the configurable options.